src/sentry/demo/data_population.py

```diff
@@ -448,148 +459,290 @@ def fix_breadrumbs(event_json, quick):
+                "span_id": frontend_root_span_id,
+            }
+        }
+
+        # React transaction
+        local_event = copy.deepcopy(react_transaction)
+        local_event.update(
+            project=react_project,
+            platform=react_project.platform,
+            event_id=uuid4().hex,
+            user=transaction_user,
+            release=release_sha,
+            timestamp=timestamp,
+            # start_timestamp decreases based on day so that there's a trend
+            start_timestamp=timestamp - timedelta(seconds=frontend_duration),
+            measurements=gen_measurements(day),
+            contexts=frontend_context,
+        )
+
+        fix_transaction_event(local_event, old_span_id)
+        safe_send_event(local_event, quick)
+
+        # note picking the 0th span is arbitrary
+        backend_parent_id = local_event["spans"][0]["span_id"]
+
+        # React error
+        local_event = copy.deepcopy(react_error)
+        local_event.update(
+            project=react_project,
+            platform=react_project.platform,
+            timestamp=timestamp,
+            user=transaction_user,
+            release=release_sha,
+            contexts=frontend_context,
+        )
+        fix_error_event(local_event, quick)
+        safe_send_event(local_event, quick)
+
+        # python transaction
+        old_span_id = python_transaction["contexts"]["trace"]["span_id"]
+        backend_duration = random_normal(1500 + 50 * day, 250, 500)
+
+        backend_context = {
+            "trace": {
+                "type": "trace",
+                "trace_id": trace_id,
+                "span_id": uuid4().hex[:16],
+                "parent_span_id": backend_parent_id,
+            }
+        }
+
+        local_event = copy.deepcopy(python_transaction)
+        local_event.update(
+            project=python_project,
+            platform=python_project.platform,
+            timestamp=timestamp,
+            start_timestamp=timestamp - timedelta(milliseconds=backend_duration),
+            user=transaction_user,
+            release=release_sha,
+            contexts=backend_context,
+        )
+        fix_transaction_event(local_event, old_span_id)
+        safe_send_event(local_event, quick)
+
+        # python error
+        local_event = copy.deepcopy(python_error)
+        local_event.update(
+            project=python_project,
+            platform=python_project.platform,
+            timestamp=timestamp,
+            user=transaction_user,
+            release=release_sha,
+            contexts=backend_context,
+        )
+        fix_error_event(local_event, quick)
+        safe_send_event(local_event, quick)
     logger.info("populate_connected_event_scenario_1.finished", extra=log_extra)
+
+
+def populate_connected_event_scenario_2(
+    react_project: Project, python_project: Project, quick=False
+):
+    """
+    This function populates a set of four related events with the same trace id:
```

Comment:
This should be
```suggestion
    This function populates a set of two related events with the same trace id:
```
Right? Or am I missing something